package.json

```diff
@@ -13,7 +13,8 @@
     "publish-release": "node scripts/publish-release"
   },
   "devDependencies": {
-    "conventional-changelog-conventionalcommits": "~4.6.0",
+    "conventional-changelog-conventionalcommits": "^4.6.1",
```

Comment:
Revert the changes of this file please, we only need the ones in the lock files.!